playbooks/develop/includes/setup_bench.yml

```diff
@@ -16,6 +16,18 @@
       creates: "{{ bench_path }}/sites/common_site_config.json"
       chdir: "{{ bench_path }}"
 
+  # In case we are re-running the script, we would like to skip the site creation
+  - name: Check whether a site exists
+    stat: path="{{ bench_path }}/sites/site1.local"
+    register: site
+
+  - name: Create new site
```

Comment:
Wrong place to add this!!! This should be part of production setup and not develop setup